core-blocks/button/index.js

```diff
@@ -76,17 +72,16 @@ export const settings = {
 
 	attributes: blockAttributes,
 
-	getEditWrapperProps( attributes ) {
-		const { align } = attributes;
+	getEditWrapperProps() {
 		const props = { 'data-resized': true };
 
-		if ( 'left' === align || 'right' === align || 'center' === align ) {
-			props[ 'data-align' ] = align;
-		}
-
 		return props;
 	},
 
+	supports: {
+		align: true,
```

Comment:
`wideAlign` should be set to `false`. Previously it was possible to set only `left`, `right` and `center`.

Reply:
Added. Thank you @gziolo ❤️